akka-docs/src/main/paradox/stream/operators/Source-or-Flow/conflate.md

```diff
@@ -18,6 +18,15 @@ Allow for a slower downstream by passing incoming elements and a summary into an
 there is backpressure. The summary value must be of the same type as the incoming elements, for example the sum or
 average of incoming numbers, if aggregation should lead to a different type `conflateWithSeed` can be used:
 
+## Example
+
+Scala
+:   @@snip [IntegrationDocSpec.scala]($code$/scala/docs/stream/IntegrationDocSpec.scala) { #conflate }
+
+Java
+:   @@snip [IntegrationDocTest.java]($code$/java/jdocs/stream/IntegrationDocTest.java) { #conflate }
+
+## Reactive Streams semantics 
```

Comment:
I see we have the 'semantics' section after the example in other places as well, but wouldn't it make sense to have it above?

Reply:
My argument is that they're less helpful, people jump here to get an example, then can dig in deeper

Reply:
whatever, consistency is king, so then change all

Reply:
Keeping the consistent one then; as-is